content/tutorial/03-sveltekit/04-headers-and-cookies/02-cookies/README.md

```diff
@@ -17,7 +17,7 @@ export function load(+++{ cookies }+++) {
 }
 ```
 
-To set a cookie, you `cookies.set(name, value, options)`. It's strongly recommended that you explicitly configure the `path` when setting a cookie, since browsers' default behaviour — somewhat uselessly — is to set the cookie on the parent of the current path.
+To set a cookie, you call `cookies.set(name, value, options)`. It's strongly recommended that you explicitly configure the `path` when setting a cookie, since browsers' default behaviour — somewhat uselessly — is to set the cookie on the parent of the current path.
```

Comment:
```suggestion
To set a cookie, call `cookies.set(name, value, options)`. It's strongly recommended that you explicitly configure the `path` when setting a cookie, since browsers' default behaviour — somewhat uselessly — is to set the cookie on the parent of the current path.
```